.github/workflows/test.yml

```diff
@@ -30,4 +30,8 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
 
-      - run: go test ./... -v -race -count=1
+      - name: Install gotestsum
+        run: go install gotest.tools/gotestsum@v1.11.0
+
+      - name: Run gotestsum
+        run: gotestsum -f testname -- ./... -race
```

Comment:
You are missing  `-count=1`

Reply:
I can add it back in, but it's already going to run every test exactly once, so it would make no difference.